
FAIL forward() and intercept() with a fulfilled promise assert_equals: expected "#1" but got ""

